python/dask_cudf/dask_cudf/io/tests/test_csv.py

```diff
@@ -170,3 +170,24 @@ def test_csv_reader_usecols(tmp_path, dtype):
     ddf2 = dask_cudf.read_csv(csv_path, usecols=["b", "c"], dtype=dtype)
 
     dd.assert_eq(ddf, ddf2, check_divisions=False, check_index=False)
+
+
+def test_read_csv_skiprows(tmp_path):
```

Comment:
Should we add tests for the failure cases?

Reply:
Done, added tests 👍